pkgs/top-level/all-packages.nix

```diff
@@ -21231,9 +21231,10 @@ with pkgs;
 
   inherit (callPackages ../development/libraries/libressl { })
     libressl_3_4
-    libressl_3_5;
+    libressl_3_5
+    libressl_3_6;
 
-  libressl = libressl_3_5;
+  libressl = libressl_3_6;
```

Comment:
Not sure about this, because the release notes say:

> This is a development release for the 3.6.x branch, and we appreciate additional testing
and feedback before the final release coming soon with OpenBSD 7.2.

Maybe it’s best to keep the default at 3.5 until the first stable release in 3.6.

Reply:
This does manage to build every dependent package and this _is_ for "unstable".

I'm probably more afraid of us forgetting to bump the default version before 22.11 and having to support/patch 3.5 for another 6 months.

Reply:
Fair point, maybe we should update to 3.6 already then. 3.5 will still be supported upstream for half a year, but 3.4 will not, should we remove 3.4 before 22.11? I tried to build `fdvPackages` with 3.6 which failed, and `ovftool` which I couldn’t test because it requires downloading some proprietary binary externally. The one other package that explicitly uses 3.4 is `wasm-pack`, which can use 3.5. I’ll open a pull request to update it.